doc/guide/cockpit-session.xml

```diff
@@ -58,6 +58,32 @@ promise.then(user => { ... });
     </variablelist>
 
     <para>Returns a promise that completes once the user information is available.</para>
+    <warning>
+	    <para>
+		    <code>cockpit.user()</code> is soft-deprecated since Cockpit 336, <link
+		    linkend="cockpit-init">cockpit.init()</link> can be used
+		    instead to obtain the user on initial page load.
+
+		    If your page needs to maintain compatibility with
```

Comment:
Text width is too narrow here...

Reply:
I just use vim's auto indention so I don't have to manually indent, and well it doesn't work for docbook